README.md

```diff
@@ -219,3 +219,10 @@ other frameworks could adopt Puppeteer as their foundational layer.
 The past few months have brought [several new libraries for automating headless Chrome](https://medium.com/@kensoh/chromeless-chrominator-chromy-navalia-lambdium-ghostjs-autogcd-ef34bcd26907). As the team authoring the underlying DevTools Protocol, we're excited to witness and support this flourishing ecosystem.
 
 We've reached out to a number of these projects to see if there are opportunities for collaboration, and we're happy to do what we can to help.
+
+#### Q: What features does Puppeteer not support?
+
+You may find that Puppeteer does not behave as expected when controlling pages that incorporate audio and video. (For example, [video playback/screenshots is likely to fail](https://github.com/GoogleChrome/puppeteer/issues/291).) There are two reasons for this:
```

Comment:
Sorry to be picky. I think we can ditch the link now b/c it's discussing lack of mp3/4 support, which is not true anymore :)

Reply:
Is MP4 supposed to work now with GoogleChrome/master? #291 now fails in a slightly different way (no error message), but the video still doesn't play.

Reply:
Indeed, looks like mp4 is still not supported by chromium atm.